### PR TITLE
Add manylinux2014 Dockerfile for CPU builds

### PR DESCRIPTION
**Pull request by sub-mod**
This PR adds manylinux2014 Dockerfile and the changes needed to install rpm packages.
Summary of changes:
- `sudo` command was removed from `install_golang.sh`.
- added `Dockerfile.manylinux2014.cpu`.
- added `install_rpm_packages.sh`.
- modified `install_pip_packages.sh` to use python36 in centos7 Image.

### CODE REVIEW

**Comment by sub-mod** [comment]
@perfinion @gunan  any feedback on this ?

File: images/Dockerfile.manylinux2014.cpu [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+LABEL maintainer="Subin Modeel <smodeel@redhat.com>"
```
Comment (sub-mod) [inline images/Dockerfile.manylinux2014.cpu]: I should probably remove this. And replace with a TF build team alias email id.

**Comment by gunan** [comment]
Any updates/requests on this?

**Comment by angerson** [comment]
Yikes, sorry! The image is building fine for me. What should I run inside the new image to build TF, so I can verify it works? Can we add that to documentation somewhere?

**Comment by sub-mod** [comment]
```
./configure
bazel build ...
```
@angerson the above commands should work. But let me test again.It's been a while.

**Comment by sub-mod** [comment]
@angerson @gunan Added README file with steps to build manylinux2014 from the new Dockerfile.
```source scl_source enable devtoolset-7 rh-python36
git clone --branch=r2.1 --depth=1 https://github.com/tensorflow/tensorflow.git
cd tensorflow/
./configure
bazel build -c opt --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" --verbose_failures //tensorflow/tools/pip_package:build_pip_package```

File: images/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Dockerfiles for TensorFlow builds
+
+## manylinux2014 builds
+Run the below commands to create manylinux2014 build environment.    
+```
+docker build -t build/manylinux2014 -f Dockerfile.manylinux2014.cpu .
+docker run -it build/manylinux2014:latest /bin/bash
+
```
Comment (angerson) [inline images/README.md:8]: Needs one more ``` here

File: images/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Dockerfiles for TensorFlow builds
+
+## manylinux2014 builds
+Run the below commands to create manylinux2014 build environment.
```
Comment (angerson) [inline images/README.md:4]: "create a"

**Comment by angerson** [comment]
I forgot tensorflow/build doesn't have an automated merging system like tensorflow. Thanks for this!

**Pull request merged**